--- skinflatplus.c.orig	2014-03-19 15:58:18.468626577 +0100
+++ skinflatplus.c	2014-03-19 16:01:01.984064840 +0100
@@ -82,12 +82,13 @@
 }
 
 bool cPluginFlat::Start(void) {
+#ifdef TEST_TRUE_COLOR
     if (!cOsdProvider::SupportsTrueColor()) {
         esyslog("skinflatplus: No TrueColor OSD found! Aborting!");
         return false;
     } else
         dsyslog("skinflatplus: TrueColor OSD found");
-    
+#endif    
     imgCache.Create();
     imgCache.PreLoadImage();
 
